back/spec/policies/initiative_policy_spec.rb

```diff
@@ -256,21 +320,40 @@
         it { is_expected.not_to permit(:create) }
         it { is_expected.not_to permit(:update) }
         it { is_expected.not_to permit(:destroy) }
+        it { is_expected.not_to permit(:accept_invite) }
 
         it 'does not index the initiative' do
           expect(scope.resolve.size).to eq 1
         end
       end
 
-      # Author can see initiative, but not edit, if review feature is active.
+      # Author can see initiative, but not edit, if review feature is active,
+      # and when status in InitiativeStatus::NOT_REVIEW_CODES
       context 'for a user who is author of the initiative' do
         let(:user) { author }
 
         it { is_expected.to permit(:show) }
         it { is_expected.to permit(:by_slug) }
         it { is_expected.to permit(:create) }
-        it { is_expected.to permit(:update) }
+        it { is_expected.to permit(:update) } # TODO: This is confusing, as editing_locked should be true when status proposed!
```

Comment:
Even though we have separate tests of `editing_locked` in this spec file, I don't like the fact that the other tests of `update` (e.g. here) often seem to contradict the expected behaviour. They don't really contradict, because we have not set the `editing_locked` values for the `initiatives` we use in examples like these, but maybe we should.

For now, I think it's OK to leave this for later, but we should return to this to not leave the tests as confusing documentation of the expected behaviour of `update`.